### PR TITLE
refactor: normalization checks

### PR DESCRIPTION
**Pull request by 121watts**
RFC https://github.com/influxdata/idpe/issues/5591

### CODE REVIEW

File: ui/src/alerting/constants/index.ts [hunk 1]
```diff
@@ -79,11 +79,11 @@ export const DEFAULT_ENDPOINT_URLS = {
 export const NEW_ENDPOINT_DRAFT: NotificationEndpoint = {
   name: 'Name this Endpoint',
   description: '',
-  status: 'active',
+  activeStatus: 'active',
   type: 'slack',
   token: '',
   url: DEFAULT_ENDPOINT_URLS['slack'],
-  loadingStatus: RemoteDataState.Done,
+  status: RemoteDataState.Done,
```
Comment (ebb-tide) [inline ui/src/alerting/constants/index.ts]: oof- this is going to be super confusing someday.  but maybe types will make it ok?

File: ui/src/alerting/utils/customCheck.test.ts [hunk 2]
```diff
@@ -9,8 +9,8 @@ const bc1: BuilderConfig = {
 }
 
 const ab1: AlertBuilderState = {
-  status: 'active',
-  checkStatus: RemoteDataState.Done,
+  activeStatus: 'active',
```
Comment (ebb-tide) [inline ui/src/alerting/utils/customCheck.test.ts]: much more clear thank you!

File: ui/src/schemas/index.ts [hunk 3]
```diff
@@ -77,6 +81,26 @@ export const cell = new schema.Entity(
...
+
+// Defines the schema for the "checks" resource
+export const checkSchema = new schema.Entity(
+  ResourceType.Checks,
+  {},
+  {
+    processStrategy: (check: GenCheck): Check => {
+      return {
+        ...check,
+        status: RemoteDataState.Done,
```
Comment (ebb-tide) [inline ui/src/schemas/index.ts]:
```suggestion
        status: RemoteDataState.Done,
        tags: check.tags || [], 
```
This was giving us problems before. Is this appropriate ?

File: ui/src/alerting/constants/index.ts (hunk 1 above)
Comment (121watts) [reply ui/src/alerting/constants/index.ts]: yea, it was definitely giving me a headache.  but I think the `activeStatus` and `status` with proper typing make it much more clear.

File: ui/src/schemas/index.ts (hunk 3 above)
Comment (121watts) [reply ui/src/schemas/index.ts]: i think i got a runtime error from this not being there.

**Pull request merged**